### ISSUE TITLE
Can I use my own face for Semantic Face Editing

### ISSUE THREAD

**Issue by Niraj-Lunavat** [genforce/interfacegan#104]
Hi there!

Thank you so much for such wonderful repo, appreciate your efforts!

Can you guys tell me, How can provide my own input images for experimentation, instead of some celeb data?
@ShenYujun @younesbelkada @clementapa 

In order to do it, What are the changes I need to make in colab file? I am using this file: https://colab.research.google.com/github/genforce/interfacegan/blob/master/docs/InterFaceGAN.ipynb#scrollTo=ccONBF60mVir

Basically, I want to use this code to check it on my set of images. 

Any help be appreciated, Thanks in advance!!

**Comment by sonichen** [comment]
maybe you can convert your image to latent code and then apply interfacegan to manipulate the latent code

**Comment by ammarft-ai** [comment]
How can I convert my image into latent code, the repo doesn't have any functionality of that sort.
